controllers/apply/form-router-next/joi-extensions/budget-items.js

```diff
@@ -44,7 +44,10 @@ module.exports = function budgetItems(joi) {
                     maxBudget: joi.number().required()
                 },
                 validate(params, value, state, options) {
-                    const total = sumBy(value, item => item.cost);
+                    const total = sumBy(
+                        value,
+                        item => parseInt(item.cost) || 0
```

Comment:
parseInt needs a radix of `10` as the second argument. Same for budget-total-costs.js